.cirrus.yaml

```diff
@@ -129,11 +129,35 @@ trigger_release_task:
     ARTIFACTORY_URL: https://repox.jfrog.io/artifactory/
     ARTIFACTORY_DEPLOY_USERNAME: repox-qa-deployer
     ARTIFACTORY_DEPLOY_PASSWORD: ENCRYPTED[!d484e19f33c9ce63b165f70e414a33b1ac6c215a126791aacbf8059626caf0fd8a78e999a20af5c1a4ba01c0b0247921!]
+    SLACK_SQ_DEVOPS_WEBHOOK: ENCRYPTED[b8ae6e031a9d9045d6314ee7ad7df1cbc57e1c3860acabe2913290329ac898dd2cf5b5093962606b28b305478c99c03f]
   gh_action_script: |
     source cirrus-env RELEASE
     CHARTS=$(find $CIRRUS_WORKING_DIR -maxdepth 1 -name "*.tgz*" -type f -exec basename "{}" ";")
     [[ "x$CHARTS" == "x" ]] && exit 0
     gh workflow run release.yml -f version=$CIRRUS_BRANCH -f buildNumber=$BUILD_NUMBER
   depends_on:
     - chart_packaging
-    - push_to_repox
\ No newline at end of file
+    - push_to_repox
+  on_failure:
+    slack_notification_script:
+      - .slack-notification.sh
+
+test_task:
+  timout_in: 15m
+  gke_container:
+    <<: *GKE_STD_CONTAINER_TEMPLATE
+    cpu: 1
+    memory: 1Gb
+  stateful: 'true'
+  environment_cache:
+    folder: ${CIRRUS_WORKING_DIR}/*.tgz*
+    fingerprint_script: echo $CIRRUS_BUILD_ID
+  env:
+    GITHUB_TOKEN: ENCRYPTED[!f458126aa9ed2ac526f220c5acb51dd9cc255726b34761a56fc78d4294c11089502a882888cef0ca7dd4085e72e611a5!]
```

Comment:
please remove the unneeded envs when you are done :)

Reply:
This whole task will be gone soon